### ISSUE TITLE
How can we use it for multi classification 

### ISSUE THREAD

**Issue by pradAI** [bellonemauro/SVM_pcl_demo#1]
Hi Mauro,

Thanks for the code available for public. 
I have a question regarding multiple classes, can we use it for classify the multiple classes.

**Comment by bellonemauro** [comment]
yes, it is possible to use it for multiple classes, but you need to set labels and dataset
